cookiecutter/{{cookiecutter.github_repo_name}}/pyproject.toml

```diff
@@ -91,7 +91,7 @@ dev-pylint = [
 ]
 dev-pytest = [
   "pytest == 7.4.0",
-  "frequenz-repo-config[extra-lint-examples] == 0.5.1",
+  "frequenz-repo-config[extra-lint-examples] == 0.5.2",
 {%- if cookiecutter.type != "api" %}
   "pytest-mock == 3.11.1",
   "pytest-asyncio == 0.21.1",
```

Comment:
Should this need to be remove in `Bump version in docs to v0.5.2`? Just checking...

Reply:
No, why?

Reply:
I got confused in the way github colored these lines and I wrongly though they were removed